tests/test_models.py

```diff
@@ -11,29 +11,25 @@
 from uuid import uuid4
 
 import pytest
-import sqlalchemy
-from mock import patch
```

Comment:
Can you look also at `conftest.py` that probably does not need mock patch either?